package.json

```diff
@@ -62,5 +62,5 @@
     "grunt-conventional-changelog": "~1.1.0",
     "grunt-ngdocs": "~0.2.5"
   },
-  "main": "index.js"
+  "main": "release/angular-ui-router.js"
```

Comment:
should it be `"release/angular-ui-router.js"` or `"./release/angular-ui-router.js"`? Does it make a difference? Bower has it one way and component.json and package.json the other.